src/Components/Viewer/ProfileMediaViewer.js

```diff
@@ -340,49 +341,51 @@ class ProfileMediaViewer extends React.Component {
         const { big: file } = photo;
 
         return (
-            <div className='media-viewer'>
-                {deleteConfirmation}
-                <div className='media-viewer-wrapper' onClick={this.handlePrevious}>
-                    <div className='media-viewer-left-column'>
-                        <div className='media-viewer-button-placeholder' />
-                        <MediaViewerButton disabled={!hasNextMedia} grow onClick={this.handleNext}>
-                            <NavigateBeforeIcon fontSize='large' style={navigationIconStyle} />
-                        </MediaViewerButton>
+            <Dialog open>
```

Comment:
This is funny: diff looks like a massive update, but it was only wrapping content into `<Dialog>...</Dialog>`.

Reply:
The option  would be useful here https://take.ms/ZWek5 😉